notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        TRAIN_VERSION = "tf-gpu.{}".format(TF)
</pre><p>Are we sure that it shouldn't be <code>tf2-gpu.{}</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>I double checked the pre-built container site (https://cloud.google.com/vertex-ai/docs/training/pre-built-containers) and make sure that the image urls are correct.</p>